awx/main/scheduler/dag_simple.py

```diff
@@ -140,36 +140,36 @@ def add_edge(self, from_obj, to_obj, label):
     def find_ord(self, obj):
         return self.node_obj_to_node_index.get(obj, None)
 
-    def _get_dependencies_by_label(self, node_index, label):
+    def _get_children_by_label(self, node_index, label):
```

Comment:
is this a rename for clarity, or something else?

Reply:
yep, for clarity